### ISSUE TITLE
open3d visualization gui how to visualize chinese?

### ISSUE THREAD

**Issue by kaixin-bai** [intel-isl/Open3D#2600]
`gui.Application.instance.create_window` can visualize chinese string, but `CollapsableVert` and `Button` will visualize chinese character as `?`. Is there any solution? Thanks!

**Comment by prewettg** [comment]
Internationalization is not currently possible, although it is on our list of things that we would like. Unfortunately, text and 3D graphics are difficult to do together, because you end up needing to have every character you will use in a texture somewhere. This is especially troublesome for languages like Chinese that have many characters in common usage. Also, the library that we are using to draw the UI, Dear Imgui, does not have full support for internationalization, which may further complicate things. (The title works, though, because the OS is drawing that)

**Comment by kaixin-bai** [comment]
Thanks for reply!

**Comment by kaixin-bai** [comment]
in ImGui it is possible to use Chinese with the following methods, it will be wonderful if you have the plan to add the function in the future.
```
// for 2500 chinese characters
fonts->GetGlyphRangesChineseSimplifiedCommon()
// for all chinese characters
fonts->GetGlyphRangesChineseFull()
```

**Comment by prewettg** [comment]
Thanks for the reference to the functions, I looked into it and it's pretty straightforward, so I've implemented it in PR #2655. Because of the size of the bitmap, and because we can't ship a 50 MB font with every character set, you'll need to specify a font to use. I've added an example in the PR at `examples/python/gui/non-english.py`; the C++ is pretty much identical.
